### PR TITLE
[MODCON - 71, 65] - Covered karate test for all sharing settings functionality

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://issues.folio.org/browse/MODCON-71
https://issues.folio.org/browse/MODCON-65
https://issues.folio.org/browse/MODCON-80
## Approach
- Covered all functionality of sharing settings 
- Moved mod-consortia package to module

### CODE REVIEW

**Comment by SerhiiNosko** [comment]
@azizbekxm please update teams-assignment file in the root folder as well?

File: mod-consortia/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SerhiiNosko) [inline mod-consortia/pom.xml:1]: @azizbekxm please update teams-assignment file in the root folder as well

File: mod-consortia/src/main/resources/thunderjet/mod-consortia/features/sharing-setting.feature [hunk 2]
```diff
@@ -0,0 +1,250 @@
+Feature: Consortia Sharing Instances api tests
```
Comment (bekhzodkurbonboev) [inline mod-consortia/src/main/resources/thunderjet/mod-consortia/features/sharing-setting.feature:1]: *Instances -> Settings

File: mod-consortia/pom.xml (hunk 1 above)
Comment (azizbekxm) [reply mod-consortia/pom.xml:1]: Updated

File: mod-consortia/src/main/resources/thunderjet/mod-consortia/features/sharing-setting.feature (hunk 2 above)
Comment (azizbekxm) [reply mod-consortia/src/main/resources/thunderjet/mod-consortia/features/sharing-setting.feature:1]: Fixed

File: mod-consortia/src/main/resources/karate-config.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
+function fn() {
+
+  karate.configure('logPrettyRequest', true);
+  karate.configure('logPrettyResponse', true);
+
+  var env = karate.env;
+
+  // The "testTenant" property could be specified during test runs
+  var testTenant = karate.properties['testTenant'];
```
[the remark this answers is not in the document]
Comment (azizbekxm) [reply mod-consortia/src/main/resources/karate-config.js:9]: Removed

File: mod-consortia/src/main/resources/karate-config.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    config.ftpUser = 'folio';
+    config.ftpPassword = 'Ffx29%pu';
+    config.prototypeTenant= 'diku'
+    config.admin = {
+      tenant: 'diku',
+      name: 'diku_admin',
+      password: 'admin'
+    }
+  } else if (env == 'thunderjet-consortia-rancher') {
+    config.baseUrl = 'https://folio-dev-thunderjet-okapi.ci.folio.org:443';
```
Comment (bekhzodkurbonboev) [inline mod-consortia/src/main/resources/karate-config.js:115]: This is the same as line 112. Also for admin credentials is not up to date, maybe it is better to get them from env as they maybe changed in future (maybe better not to expose them).

File: mod-consortia/src/main/resources/karate-config.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+  }
+  karate.repeat(100, rand);
+
+  if (env == 'snapshot-2') {
+    config.baseUrl = 'https://folio-snapshot-2-okapi.dev.folio.org:443';
+    config.edgeUrl = 'https://folio-snapshot-2.dev.folio.org:8000';
+    config.ftpUrl = 'ftp://ftp.ci.folio.org';
+    config.ftpPort = 21;
+    config.ftpUser = 'folio';
+    config.ftpPassword = 'Ffx29%pu';
```
Comment (SerhiiNosko) [inline mod-consortia/src/main/resources/karate-config.js:83]: these ftp properties not yet removed

File: mod-consortia/src/main/resources/karate-config.js (hunk 4 above)
Comment (azizbekxm) [reply mod-consortia/src/main/resources/karate-config.js:115]: changed to variable

File: mod-consortia/src/main/resources/karate-config.js (hunk 5 above)
Comment (azizbekxm) [reply mod-consortia/src/main/resources/karate-config.js:83]: removed

**Review by SerhiiNosko — APPROVED** [review]

**Review by bekhzodkurbonboev — APPROVED** [review]

**Pull request merged**